gradle.properties

```
@@ -18,9 +18,9 @@ reactorCoreVersion = 3.3.8.RELEASE
 reactorKotlinVersion = 1.0.2.RELEASE
 
 # JDA
-jdaVersion = 5.0.0-beta.24
+jdaVersion = 5.0.1
 jdaChewtilsVersion = 2.0-SNAPSHOT
```

Probably worth switching to the non-snapshot version now:
```suggestion
jdaChewtilsVersion = 2.0
```

You'll also need to [swap to the chewtils releases repo](https://github.com/discord-diabetes/diabot/blob/1be2982d0c6e9ec1a783168594237d218a115a76/build.gradle#L25):
```groovy
maven { url "https://m2.chew.pro/releases" }
```